src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporter.cs

```diff
@@ -26,7 +26,7 @@ public class OtlpTraceExporter : BaseExporter<Activity>
     /// </summary>
     /// <param name="options">Configuration options for the export.</param>
     public OtlpTraceExporter(OtlpExporterOptions options)
-        : this(options, new(), null)
+        : this(options, sdkLimitOptions: new())
```

Comment:
Just to be consistent with OtlpMetricExporter and OtlpLogExporter.

```suggestion
        : this(options, sdkLimitOptions: new(), transmissionHandler: null)
```